### DISCUSSION TITLE
How did you setup a page at /docs route

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#906] [Q&A] [answered]
I am trying to set a page at /docs route but it is showing various erros, I have reffered the way you have built the fumadocs website but it is not working if I try the same approach. 

How can I set up a page instead of a mdx file at /docs

**Comment by fuma-nama** [comment, marked as answer]
See https://github.com/fuma-nama/fumadocs/discussions/892#discussioncomment-10744608
